### DISCUSSION TITLE
One config per notebook (or "profiles")

### DISCUSSION THREAD

**Discussion by mbaz** [zim-desktop-wiki/zim-desktop-wiki#2714] [Q&A] [answered]
I would like to have two Zim notebooks, one to be used as a journal and the other as a task list. I'd like each notebook to have a separate configuration (to avoid, for example, seeing a task list pane in my journal or a ToC in my task list). I haven't found a way to do this -- any configuration change in one notebook is seen by the other.

Is there a way to have two independently-configured notebooks?

**Comment by jaap-karssenberg** [comment, marked as answer]
Only way to do so if to start from a shell script and set the XDG
environment parameters.

Op di 31 dec. 2024 16:54 schreef mbaz ***@***.***>:

**Comment by mbaz** [comment]
Thank you! This solution seems to work fine (so far). For comleteness: assuming the second wiki will be stored under `~/newwiki`, I created `~/newwiki/.config`. To keep my GTK theme, I also ran `cd ~/newwiki/.config && ln -s ~/.config/gtk-3.0 && cd`. Then, running zim with

    XDG_CONFIG_HOME=~/newwiki/.config zim

loads a separate zim "profile".
